python/interpret_community/explanation/explanation.py

```diff
@@ -338,8 +338,9 @@ def local_importance_values(self):
             the last will be for "fish". If you choose to pass in a classes array to the explainer, the names should
             be passed in using this same order.
         :rtype: list[list[float]] or list[list[list[float]]] or scipy.sparse.csr_matrix
+            or list[scipy.sparse.csr_matrix]
         """
-        if sp.sparse.issparse(self._local_importance_values):
+        if sp.sparse.issparse(self._local_importance_values) or isinstance(self._local_importance_values, list):
```

Comment:
It's very weird that we're communicating sparse multiclass based on whether `_local_importance_values` is a list, and beyond being weird it's pretty fragile in the sense that at some point regular dense `_local_importance_values` could easily be a list. Can we have a field in the explanation keeping track?

Reply:
good point, I added a property that determines if the local importance values are sparse.  I think that makes the code cleaner.